sdk/python/feast/embedded_go/online_features_service.py

```diff
@@ -132,9 +132,15 @@ def get_online_features(
         resp = record_batch_to_online_response(record_batch)
         return OnlineResponse(resp)
 
+    def start_http_server(self, host: str, port: int):
```

Comment:
We need at least some simple e2e test for this.

Reply:
Added